docker/smqtk_iqr_playground/default_confs/cpu/compute_many_descriptors.json

```diff
@@ -9,51 +9,50 @@
             "db_port": 5432,
             "db_user": "smqtk",
             "table_name": "descriptors_resnet50_pool5",
-            "type_col": "type_str",
             "uuid_col": "uid"
         },
-        "type": "smqtk.representation.descriptor_element.postgres.PostgresDescriptorElement"
-    },
+        "type": "smqtk_descriptors.impls.descriptor_element.postgres.PostgresDescriptorElement"
+        },
     "descriptor_generator": {
-        "smqtk.algorithms.descriptor_generator.caffe_descriptor.CaffeDescriptorGenerator": {
-            "batch_size": 20,
+        "smqtk_descriptors.impls.descriptor_generator.caffe1.CaffeDescriptorGenerator": {
+            "batch_size": 10,
```

Comment:
What was the motivation for the batch size change?

Reply:
Hmm wasn't intentional. I took the `descriptor_generator` block from the `runApp.IqrService.json` config file and pasted it into this config file, since `runApp.IqrService.json` was already updated to use the breakout repos.